src/main.js

```diff
@@ -18,13 +18,16 @@ const router = new VueRouter({
 Vue.use(VueRouter)
 Vue.use(VuePageTransition)
 
-new Vue({
+const VueApp = new Vue({
   router,
   store,
   vuetify,
   render: h => h(App)
-}).$mount('#app')
+})
+
+VueApp.$mount('#app')
 
 if (window.Cypress) {
```

Comment:
@vickywane is there a way to remove test framework dependencies from the tested code? Do we really need Cypress here? Instead, how about doing the environment setup as part of the test code in the test setup sections?

Reply:
Since i refactored the test, there is no need for the Cypress instance here. 

I have restored the file back to it's original state.

Reply:
Seems like you forgot to synch your local changes with remote.

Reply:
I synced it and the  changes are correct. See subsequent comments.